jena-base/src/main/java/org/apache/jena/atlas/lib/DateTimeUtils.java

```diff
@@ -18,29 +18,22 @@
 
 package org.apache.jena.atlas.lib;
 
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
 import java.util.Calendar ;
 import java.util.Date ;
 import java.util.GregorianCalendar ;
 
 import org.apache.commons.lang3.time.FastDateFormat ;
 
 public class DateTimeUtils {
-
-    // Include timezone (even xsd:dates have timezones; Calendars have timezones)
-    // NB in SimpleDateFormat != FastDateFormat
-    //  SimpleDateFormat does not format Calendars.
-    //  SimpleDateFormat has "X" for  ISO format tmezones (+00:00)  
-    //    FastDateFormat uses "ZZ" for this.
-    private static final FastDateFormat dateTimeFmt_display = FastDateFormat.getInstance("yyyy/MM/dd HH:mm:ss z") ;
-    private static final FastDateFormat dateFmt_yyyymmdd    = FastDateFormat.getInstance("yyyy-MM-ddZZ") ;
-    // For milliseconds == 0
-    private static final FastDateFormat dateTimeFmt_XSD_ms0     = FastDateFormat.getInstance("yyyy-MM-dd'T'HH:mm:ssZZ") ;
-    // For milliseconds != 0
-    private static final FastDateFormat dateTimeFmt_XSD_ms      = FastDateFormat.getInstance("yyyy-MM-dd'T'HH:mm:ss.SSSZZ") ;
-    // For milliseconds == 0
-    private static final FastDateFormat timeFmt_XSD_ms0         = FastDateFormat.getInstance("HH:mm:ssZZ") ;
-    // For milliseconds != 0
-    private static final FastDateFormat timeFmt_XSD_ms          = FastDateFormat.getInstance("HH:mm:ss.SSSZZ") ;
+    // Use xxx to get +00:00 format with DateTimeFormatter
```

Comment:
Oh, didn't know about `xxx` for the time-zone offset. Thanks!

Reply:
Nor did I until a few hours ago!